.gitignore

```diff
@@ -24,7 +24,7 @@ testbin/*
 *.swp
 *.swo
 *~
-kuadrant-operator
+/kuadrant-operator
```

Comment:
@didierofrivia  This line is for ignoring `kuadrant-operator` folder at the root of the repo ? When would there be a `kuadrant-operator` folder at the root of the repo ?

Reply:
It will match a directory _or file_ named `kaudrant-operator` at root level relative to the .gitignore, this happens when building the binary and we want to exclude it. Thing is without the `/` it was matching also the chart directory within `/charts`

Reply:
Ah ok, makes sense. Thanks for clarifying!